pkgs/by-name/kg/kgeotag/package.nix

```diff
@@ -0,0 +1,26 @@
+{ cmake, extra-cmake-modules, fetchFromGitLab, lib, libsForQt5 }:
+
+libsForQt5.mkDerivation rec {
+  pname = "kgeotag";
+  version = "1.4.0";
+
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    repo = pname;
+    owner = "graphics";
+    rev = "v${version}";
+    hash = "sha256-az/kXEhD/RGfxOMiLEcb651WVPcyBSa8B2sWOF7bz8E=";
+  };
+
+  nativeBuildInputs = [ cmake extra-cmake-modules ];
+
+  buildInputs = [ libsForQt5.libkexiv2 libsForQt5.marble ];
+
+  meta = with lib; {
+    homepage = "https://kgeotag.kde.org/";
+    description = "A stand-alone photo geotagging program";
+    changelog = "https://invent.kde.org/graphics/kgeotag/-/blob/master/CHANGELOG.rst";
+    license = licenses.gpl3;
```

Comment:
The license should be either gpl3Only or gpl3Plus unstead of gpl3. That is because gpl3 is deprecated.